.github/workflows/test-objc.yaml

```diff
@@ -172,22 +180,17 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2
       - name: Pull Carthage dependencies
-        run: carthage bootstrap
-      - name: Build Carthage projects - excluding arm64
         run: |
-          # ------ beginning of workaround
```

Comment:
I like how we switch one workaround to _another_ workaround :D

Reply:
macOS BigSur on github actions already supports Carthage 0.38. It looks like the 10.15 should be updated in a couple of days... probably we can wait a little with merging and work on other edits and remove the workaround even in this PR :)